src/split_train_test.py

```diff
@@ -0,0 +1,73 @@
+import os
+import pandas as pd
+from sklearn.model_selection import train_test_split
+from deepchecks.tabular import Dataset
+from deepchecks.tabular.checks import DatasetsSizeComparison, TrainTestSamplesMix, MultivariateDrift, LabelDrift, FeatureDrift
+
+def split_train_test_data(cleaned_data, train_data_size, stratify_by=None):
+    """Split train test data using cleaned data"""
+
+    # Ensure cleaned_data is a dataframe
+    if not isinstance(cleaned_data, pd.DataFrame):
+        raise TypeError("The cleaned_data must be a pandas DataFrame.")
+
+    # Ensure train_data_size is a valid proportion
+    if not (0 < train_data_size < 1):
+        raise ValueError("train_data_size must be a float between 0 and 1.")
+
+    # Handle stratify_by
+    if stratify_by:
+        if stratify_by not in cleaned_data.columns:
+            raise KeyError(f"The column '{stratify_by}' does not exist in the cleaned_data.")
+    
+    # create the split
+    data_train, data_test = train_test_split(
+        cleaned_data, train_size=train_data_size, stratify=stratify_by #cleaned_data["diagnosis"]
+    )
+    
+    return data_train, data_test
+
+def validate_split_data(data_train, data_test):
```

Comment:
I feel like this function should take in inputs for all these thresholds. This would make the function more reusable across projects. Maybe the input could be a dictionary?